.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,25 @@
+name: Unit Tests
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: run unit tests
+      run: |
+        npm i
+        npm test
+      env:
+        CI: true
```

Comment:
@PeterNgTr isnt that env var set by default? Most CI systems do set it by default, but I am unsure about GitHub.

Reply:
yeah, it's be default. I could try to remove it to see if anything is impacted.

Reply:
okay, nothing is impacted!